droid-hal-prjconf.inc

```diff
@@ -34,8 +34,8 @@ if [ -e $SRC_SPECIFIC/prjconf.xml ]; then
     cat $SRC_SPECIFIC/prjconf.xml >> $DEST
 fi
 
-cat $SRC_GENERIC/macros.xml >> $DEST
 if [ -e $SRC_SPECIFIC/macros.xml ]; then
+    cat $SRC_GENERIC/macros.xml >> $DEST
```

Comment:
I would almost do `echo "Macros:" >> $DEST` instead of this one, that way nobody adds anything to the macros.xml file which would not be used when there is no source specific things.

Reply:
Yes, as we don't have any generic macros. I'll do that.